components/MediList/index.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { dummyMedications } from '../../dummydata';
+
+export default function MediList() {
+  const [medikament, setMedikament] = useState('');
+  const [dosage, setDosage] = useState('');
+
+  const handleAddMedikament = (e) => {
+    e.preventDefault();
+    const newMedikament = {
+      id: dummyMedications.length + 1,
+      medicationName: medikament,
+      dosage: dosage,
+    };
+    console.log('Neues Medikament:', newMedikament);
+    setMedikament('');
+    setDosage('');
+  };
+
+  return (
+    <Container>
+      <Form onSubmit={handleAddMedikament}>
+        <input
+          type="text"
+          placeholder="Medikament eingeben"
+          value={medikament}
+          onChange={(e) => setMedikament(e.target.value)}
+          required
+        />
+        <input
+          type="text"
+          placeholder="Dosierung eingeben"
+          value={dosage}
+          onChange={(e) => setDosage(e.target.value)}
+          required
+        />
```

Comment:
I see no explicit form submission trigger. I would recommend to add a submit button within the form element.